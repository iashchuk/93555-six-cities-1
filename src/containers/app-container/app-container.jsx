import React from "react";
import { connect } from "react-redux";

import App from "../../components/app/app.jsx";

export const AppContainer = (props) => {
  return <App {...props} />;
};

const mapStateToProps = ({ auth }) => {
  return {
    isAuthenticated: auth.isAuthenticated
  };
};

export default connect(mapStateToProps)(AppContainer);
