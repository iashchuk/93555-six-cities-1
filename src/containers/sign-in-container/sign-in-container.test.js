import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { AppContainer } from "./app-container.jsx";

configure({ adapter: new Adapter() });

describe(`AppContainer`, () => {
  it(`renders correctly`, () => {
    const component = shallow(<AppContainer />);
    expect(component).toMatchSnapshot();
  });
});
