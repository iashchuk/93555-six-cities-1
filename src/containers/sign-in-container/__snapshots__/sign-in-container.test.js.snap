// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInContainer renders correctly 1`] = `ShallowWrapper {}`;
